### ISSUE TITLE
[Bug]: Some fluid textures go missing

### ISSUE THREAD

**Issue by embeddedt** [FalsePattern/AnimFix#4]
### OS

Linux/BSD

### GPU

NVIDIA

### Modpack (Optional)

Private modpack

### Game log

[fml-client-latest.log](https://github.com/FalsePattern/AnimFix/files/8886540/fml-client-latest.log)

### Description

When AnimFix is installed in my private modpack, the still textures of some fluids turn into the missing texture. Reducing the number of mods seems to make the issue gradually go away for some fluids, so it may be related to having high numbers of fluid registered. Removing AnimFix but leaving all the other mods fixes the issue for every fluid.

I do have EndlessIDs installed as well, but fluids with a block ID below 4095 are still affected.

I'm aware that this isn't a lot of information to go by, but was hoping you might have some sort of idea where the issue might be/where I could do some more debugging.

**Comment by FalsePattern** [comment]
I upgraded some ancient code i used for the "megatexture" remapping step, i feel like this should fix it.

I don't want to spam github releases, so i just pushed it directly to my maven: https://maven.falsepattern.com/releases/com/falsepattern/animfix-mc1.7.10/1.4.0-8-gec668a6/animfix-mc1.7.10-1.4.0-8-gec668a6.jar, please check it and report back whether it fixed this problem or not.

**Comment by embeddedt** [comment]
This version crashes with the following error. Is it the dev version instead of the SRG one?

```
[16:48:52] [Client thread/FATAL] [mixin]: Mixin apply failed mixins.animfix.json:client.minecraft.TextureMapMixin -> net.minecraft.client.renderer.texture.TextureMap: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException Shadow field mipmapLevels was not located in the target class net.minecraft.client.renderer.texture.TextureMap. Using refmap mixins.animfix.refmap.json
org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: Shadow field mipmapLevels was not located in the target class net.minecraft.client.renderer.texture.TextureMap. Using refmap mixins.animfix.refmap.json
```

**Comment by FalsePattern** [comment]
huh, that's strange... i'll just create a github release then and mark it as a pre-release, it looks like i have something misconfigured

https://github.com/FalsePattern/AnimFix/releases/tag/1.4.1-pre01

**Comment by embeddedt** [comment]
Looks like this is indeed fixed in 1.4.1-pre01, thanks!

**Comment by embeddedt** [comment]
@FalsePattern Sorry, I spoke too soon. Your fix worked for fluids but I just went into an ocean monument and found solid white prismarine. After discovering that prismarine uses animated textures, I removed AnimFix and the issue went away. There might still be an edge case?
